modules/introduction/pages/whats-new.adoc

```diff
@@ -1,193 +1,108 @@
+To learn more about the Flex Index feature, see xref:n1ql:n1ql-language-reference/flex-indexes.adoc[Flex Indexes].
 
-You can use the WITH clause to create a common table expression. For more information, see xref:n1ql:n1ql-language-reference/with.adoc[WITH Clause].
+==== Index Advisor 
 
-===== Query and Search Integration
+“Index Advisor” is promoted as a supported feature, and is now available as an online tutorial. 
 
-Couchbase’s powerful full-text search capability can now be fully integrated with N1QL. This feature extends N1QL search functionality with fuzzy and non-exact text searches across multiple fields and any order. By enabling developers to use full-text search queries directly within a N1QL query, it eliminates the need to write complex code to process and combine the results from separate SQL and search queries.
 
-For more information, see xref:n1ql:n1ql-language-reference/searchfun.adoc[Search Functions].
+The Index Advisor eliminates the guesswork in index creation and recommends changes to existing indexes wherever applicable. By providing recommendations for a specific query, or a set of queries from the Couchbase server log, it helps you create optimum indexes for your queries. 
 
-===== Cost-Based Optimizer (Developer Preview)
+The Index Advisor also enables you to monitor performance and manage indexes to reflect the current usage pattern. 
 
-Couchbase delivers a cost-based optimizer, deriving access patterns based on statistical data for query tuning to the NoSQL world. This feature helps developers by eliminating the time spent on tweaking a query using a rule-based optimizer to choose the most optimal execution plan.
+Starting with this release, the Index Advisor is a supported feature and is available as an https://index-advisor.couchbase.com[online tutorial] too. 
 
-For more information, see xref:n1ql:n1ql-language-reference/cost-based-optimizer.adoc[Cost-Based Optimizer].
+==== TTL Support in Query
 
-===== Index Advisor (Developer Preview)
+Document expiration (Time to Live) is supported in Couchbase at both the bucket and the document level. The setting is intended for use on documents that can be automatically removed after a certain period of time. The expiration time is set during the document creation time using the Couchbase SDK. 
 
-Index advisor provides index suggestions to optimize query tuning and improve the performance of the queries. Running workloads are monitored and analyzed to suggest appropriate indexes for improving response times.
+With N1QL TTL, the expiration time can now be set and updated directly using the N1QL syntax. This enhancement allows administrators to change the document expiration policy after the document was created, or selectively choose documents for deletion via the document expiration mechanism.
```

Comment:
Would be good to have an xref here to the syntax.

Reply:
waiting for link. will add later